storage/storage_test.go

```diff
@@ -322,7 +335,12 @@ func testManifest(t *testing.T, factory storageFactory) {
 					Path:        fmt.Sprintf("%s.%s-%s.vsix", test.extension.Publisher, test.extension.Name, test.version),
 					Addressable: "true",
 				})
-				require.Equal(t, test.expected, manifest)
+				if f.expectedManifest != nil {
+					f.expectedManifest(test.expected)
+				}
+				if !assert.Equal(t, test.expected, manifest) {
+					fmt.Println("Asd")
```

Comment:
Is the `Asd` a placeholder message?  Also the other tests are using `require`; should this as well?

Reply:
Whoops! I just needed a line to put a breakpoint on lol. Will fix, good catch